ui/component/or-attribute-input/src/index.ts

```diff
@@ -262,6 +279,54 @@ export const jsonFormsInputTemplateProvider: (fallback: ValueInputProvider) => V
                 return jsonForms.value.checkValidity();
             }
         };
+    }else if (valueDescriptor.name === WellknownValueTypes.ANOMALYDETECTIONCONFIGURATION) {
+        const onChangedA = (dataAndErrors: { errors: ErrorObject[] | undefined, data: any },update :boolean) => {
+            const newConfig: AnomalyDetectionConfigObject = dataAndErrors.data;
+            valueChangeNotifier({
+                value: newConfig
+            });
+        }
+        const validate = async (valid:boolean) => {
+
+        }
+        const templateFunction: ValueInputTemplateFunction = (value, focused, loading, sending, error, helperText) => {
+            if(!value){
+                value ={methods:[
+                    {
+                        type: "global",
+                        name: "Method 1",
+                        onOff: true,
+                        deviation: 10,
+                        alarm: {
+                            content: "%ASSET_NAME%\n%ATTRIBUTE_NAME%\n%METHOD_TYPE%"
+                        },
+                        alarmOnOff: false,
+                        minimumDatapoints: 2,
+                        timespan: "PT20M"
+                    }]}
+            }
+            return html`
+                <style>
+                    .test {
+                        display: flex;
+                        flex-direction: column;
+                        width: 100%;
+                    }
+                </style>
```

Comment:
Style tags shouldn't be included in the render. You should use one of these solutions;
https://lit.dev/docs/v2/components/styles/

Reply:
I'm no frontend wizard but I don't see how to implement this since this html is in an template function and not a component. moved the styling in the div itself so the `test` class is also gone.